### DISCUSSION TITLE
postUpgradeTasks dependent automerge

### DISCUSSION THREAD

**Discussion by tonnico** [renovatebot/renovate#21993] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.71.4

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi,

Sometimes snapshot tests break our update. That's intentional. I want to update these snapshots automatically. But in this case, I would like to approve this change through a review. 
If there is no snapshot update I want to automerge.

Snapshot updates would be done in a [postUpgradeTasks](https://docs.renovatebot.com/configuration-options/#postupgradetasks).

Is this already somehow possible?

### What you tried so far.

-

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Do you mean that you want to disable automerge if postUpgradeTasks changed any files, but allow it if the postUpgradeTasks did not?

**Reply by tonnico** [reply]
Yes, exactly.

**Reply by rarkins** [reply]
I can't think of any way to achieve that, sorry. Perhaps you could add a github action to automate the process of requesting review or changes in order to stop the automerge.

**Reply by tonnico** [reply]
Can I create a resolvable comment in the MR with the bot?

**Reply by rarkins** [reply]
I don't think there's any way to create arbitrary comments today either

**Reply by tonnico** [reply]
Ok, thanks for your fast response.
